streamer/src/nonblocking/quic.rs

```diff
@@ -218,7 +231,24 @@ async fn run_server(
         }
 
         if let Ok(Some(connection)) = timeout_connection {
-            info!("Got a connection {:?}", connection.remote_address());
+            let remote_address = connection.remote_address();
+
+            let do_rate_limiting = true;
```

Comment:
why do we have this flag?

Reply:
I will remove -- it was used when I did the test in a/b testing to compare the differences.